MMM-OClock.js

```diff
@@ -273,27 +304,26 @@ Module.register("MMM-OClock", {
   },
 
   drawPost: function(ctx, item) {
-      if (this.config.useNail) {
-        ctx.beginPath()
-        ctx.lineWidth=1;
-        ctx.fillStyle = item.c
-        ctx.arc(item.x, item.y, (this.config.nailSize/2), 0, 2*Math.PI)
-        ctx.closePath()
-        ctx.fill()
-
-        ctx.beginPath()
-        ctx.lineWidth=1;
-        ctx.fillStyle = this.config.nailBgColor
-        ctx.arc(item.x, item.y, (this.config.nailSize/2) - 5, 0, 2*Math.PI)
-        ctx.closePath()
-        ctx.fill()
-      }
-
+    if (item.h === 'second') return;
```

Comment:
Disabling the post draw for seconds for now --  will need to clear the previous post draw since we longer clear the entire canvas.